doc/src/build/install.md

```diff
@@ -164,27 +164,26 @@ Use the following command to install Git:
 $ brew install git
 ```
 
-You can also Download and install the [Git command line interface](https://git-scm.com/download/) for your operating system.
+After installing Git, download and install the [Git command line interface](https://git-scm.com/download/).
 
 Proceed to [Install Sui binaries](#install-sui-binaries) to continue installing Sui.
 
 
 ## Windows prerequisites
 
-Install the prerequisites listed in the following section if to work with Sui on  Microsoft Windows.
+Install the following prerequisites to work with Sui on Microsoft Windows 11.
 
 ### cURL
 
-Download and install [cURL](https://curl.se) from https://curl.se/windows/.
+Windows 11 ships with a Microsoft version of [cURL](https://curl.se/windows/microsoft.html) already installed. If you want to use the curl project version instead, download and install it from [https://curl.se/windows/](https://curl.se/windows/).
 
 ### Git CLI
 
-Download and install the [Git command line interface](https://git-scm.com/download/)
-for your operating system.
+Download and install the [Git command line interface](https://git-scm.com/download/).
```

Comment:
do any of the commands actually use the Git CLI? There is one gh clone in a topic, but that can also be done with just git. Why do we need Git CLI?